modules/images-other-jenkins-memory.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins.adoc
+
+[id="images-other-jenkins-memory_{context}"]
+= Jenkins memory requirements
+
+When deployed by the provided Jenkins Ephemeral or Jenkins Persistent
+templates, the default memory limit is `512 MiB`.
```

Comment:
the default is now 1Gi in our templates

Reply:
But that default is a minimal value to facilitate more constrained, online environments.

Running with a value of 4Gi, to give the upstream jenkins recommended 2Gi java heap to jenkins, but then also allow for pipelines to fork processes (i.e. `sh ' ...'`) with an additional 2Gi of mem available outside of the java heap.